src/components/Synthetics/GmSwap/GmSwapBox/useUpdateByQueryParams.tsx

```diff
@@ -68,6 +74,19 @@ export function useUpdateByQueryParams({
         }
       }
 
+      if (pickBestGlv) {
+        setOperation(Operation.Deposit);
+
+        const glvs = Object.values(marketsInfo).filter((m) => isGlv(m)) as GlvInfo[];
```

Comment:
won't it work without `as` if you do `.filter(isGlv)`?

Reply:
Yup, removed